tests/unit-tests/test_which.py

```diff
@@ -21,7 +21,7 @@
 
 
 def test_basic():
-    assert which("ls") == "/bin/ls"
+    assert which("ls") in ["/usr/bin/ls", "/usr/sbin/ls", "/bin/ls"]
```

Comment:
This assert will fail on nix. Solution would be to check if the path which returns ends with a match in that list and don't assume the whole path is a match.